test/integration/targets/forks/test_forks.yml

```diff
@@ -0,0 +1,229 @@
+---
+- hosts: localhosts
+  gather_facts: false
+  vars:
+    forksdir: ~/ansible_testing/forks.dir/
+  tasks:
+    - name: Clean forksdir '{{ forksdir }}'
+      file:
+        state: absent
+        path: '{{ forksdir }}'
+      ignore_errors: yes
+      run_once: yes
+    - name: Create forksdir '{{ forksdir }}'
+      file:
+        state: directory
+        path: '{{ forksdir }}'
+      run_once: yes
+    - block:
+      - name: "Test 1 (max forks: 3)"
+        command: "{{ ansible_python_interpreter }}"
```

Comment:
Per @bcoca's comment, this should be `ansible_playbook_python` instead of `ansible_python_interpreter` since we are just running against localhost

Reply:
The inventory contains this:
`ansible_python_interpreter="{{ ansible_playbook_python }}"`

It is done this way in lots of inventory files in the test folder.